easy-to-use-api/klips-chart-api/src/util/Chart.ts

```diff
@@ -158,6 +157,17 @@ export const setupBaseChart = (): echarts.EChartsOption => {
       top: 100,
       bottom: 100
     },
+    graphic: [{
+      type: 'image',
+      onclick:  function () { window.open('http://www.klips-projekt.de/'); },
+      style: {
+        x: 8,
+        y: 20,
+        height: 40,
+        image: 'https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png',
```

Comment:
For the moment OK to link the KLIPS logo of the main page. In the future we should consider hosting the logo our self, to avoid having another origin when embedding the widget.